content/docs/command-reference/exp/show.md

```diff
@@ -235,8 +245,32 @@ $ dvc exp show --all-commits --include-params=featurize --sort-by=auc --sort-ord
 └───────────────────────┴──────────────┴─────────┴────────────────────────┴──────────────────┘
 ```
 
-Note that in the final example, Git commits remain in chronological order. The
+Note that in the last example, Git commits remain in chronological order. The
 sorting only applies to experiment groups (sharing a parent commit).
 
+To generate an interactive parallel coordinates plot from the table:
+
+```dvc
+$ dvc exp show --html
```

Comment:
Let's give this example a separate header?

````suggestion
## Example: Parallel coordinates plot

To generate an interactive parallel coordinates plot based on the experiments
and their parameters:

```dvc
$ dvc exp show --html
````

Reply:
But again, definition/description of what a PCP is is still missing somewhere.

Reply:
Do you think that it should be a link or we should write our own explanation of PCP ?